lib/addons/aws-privateca-issuer/index.ts

```diff
@@ -0,0 +1,103 @@
+// lib/kubevious_addon.ts
+import { Construct } from 'constructs';
+import { ManagedPolicy } from "aws-cdk-lib/aws-iam";
+import merge from "ts-deepmerge";
+import { ServiceAccount } from 'aws-cdk-lib/aws-eks';
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { dependable, setPath, createNamespace } from '../../utils';
+import { ClusterInfo, Values } from "../../spi";
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface AWSPrivateCAIssuerAddonProps extends HelmAddOnUserProps {
+
+    /**
+     * The name of the service account to use. If createServiceAccount is true, a serviceAccountName is generated.
+     */
+    serviceAccountName?: string;
+    /**
+     * An array of Managed IAM Policies which Service Account needs for IRSA Eg: irsaRoles:["AWSCertificateManagerPrivateCAFullAccess"]. If not empty
+     * Service Account will be Created by CDK with IAM Roles Mapped (IRSA). In case if its empty, Service Account will be created with out IAM Roles
+     */   
+     irsaRoles?: string[];    

```

Comment:
Let's call them what they are: iamPolicies. The fact they are attached to the role for IRSA is our implementation details. We only create a single role behind the scenes.